### ISSUE TITLE
Proper way to visualize segmentation results

### ISSUE THREAD

**Issue by huuquan1994** [suhyeonlee/WildNet#6]
Hi there,

Thanks so much for your work! I'm new to semantic segmentation and have a question!

I tried to segment some GTAV images using your ResNet-50 pre-trained model. But the final result seemed to be messy.
Here is one of the final results
![image](https://user-images.githubusercontent.com/9111958/210577387-d7b59ebf-c6ec-43de-b11b-e9e0d968cd25.png)

Here is the code I used to predict the mask (kind of pseudo-code).
```python

### Load model
net = network.get_net(args, criterion, criterion_aux, args.cont_proj_head, args.wild_cont_dict_size)

model_path = 'r50os16_gtav_wildnet.pth' # provided on Google Drive
optim, scheduler = optimizer.get_optimizer(args, net)
net = torch.nn.SyncBatchNorm.convert_sync_batchnorm(net)
epoch, mean_iu = optimizer.load_weights(net, optim, scheduler, model_path, args.restore_optimizer)

# tested on 1 GPU so no need to warp_network_in_dataparallel
net.cuda()
net.eval()

### For transforming the input
input_transform = [standard_transforms.ToTensor()]
input_transform = standard_transforms.Compose(input_transform)
rgb_mean_std = ([0.485, 0.456, 0.406], [0.229, 0.224, 0.225])

### Read GTAV image
img = Image.open(img_path).convert('RGB')
img = input_transform(img)
img = standard_transforms.Normalize(*rgb_mean_std)(img)

img = img.unsqueeze(0)
img = img.cuda()

### Predict the mask
with torch.no_grad():
    output = net(img)

pred = output.data.max(1)[1].cpu().numpy()
```

I want to visualize the prediction like the one in [here](https://github.com/suhyeonlee/WildNet/blob/main/assets/comparison.png)

Thank you in advance and hope to hear from you soon!
